### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by secondsun**
This PR removes non maven build systems, adds sources to generated packages, and updates dependencies in the maven pom. Sources have been regenerated as well. The pom updates should keep the renovate away for a bit.

### CODE REVIEW

**Comment by wtrocki** [comment]
Can you comment on folder structure. I do not get what sdk folder does and why all source code is there.

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,15 @@ yarn.lock
 *.processed
 **/target/*
 *.versionsBackup
+
+**/.gradle/*
```
Comment (wtrocki) [inline .gitignore:48]: Openapi ignore file can be used to stop generating those files

**Comment by wtrocki** [comment]
I see that we have mustache file - is that gor dependency updates?

**Comment by secondsun** [comment]
@wtrocki it is the pom template that has 1) includes the information to publish packages and 2) fixes some dependencies rennovate was complaining about

**Comment by secondsun** [comment]
@wtrocki sdk folder? You mean what the SDK's in the packages directories are and are linked in the main readme?

**Comment by wtrocki** [comment]
> Can you comment on folder structure. I do not get what sdk folder does and why all source code is there.

^^

**Review by wtrocki — APPROVED** [review]

**Pull request merged**